src/test/java/com/example/weather/client/mappers/ConditionsMapperTest.java

```diff
@@ -0,0 +1,28 @@
+package com.example.weather.client.mappers;
+
+import com.example.weather.client.models.dto.ConditionsDto;
+import org.junit.jupiter.api.Test;
+import uk.co.jemos.podam.api.PodamFactory;
+import uk.co.jemos.podam.api.PodamFactoryImpl;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ConditionsMapperTest {
+
+    private static PodamFactory FACTORY = new PodamFactoryImpl();
```

Comment:
There could be just one, reusable `PodamFactory` instance. 
Create some utility class in test sources, and create a generic method for manufacturing a pojo.